src/Router.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import Navigation from './Navigation';
+import Home from './routes/Home';
+import Chat from './routes/Chat';
+import Profile from './routes/Profile';
+import LogIn from './routes/LogIn';
+import ChatList from './routes/ChatList';
+import styled from 'styled-components';
+
+const Router = ({ isLoggedIn, user, friends }) => {
+  return (
+    <Appwrapper>
+      <BrowserRouter>
+        {isLoggedIn && <Navigation />}
+        <Switch>
+          {isLoggedIn ? (
+            <InteractingSpaceWrapper>
+              <Route exact path={`/`}>
+                <Home friends={friends} />
+              </Route>
+              <Route exact path={`/chatlist`}>
+                <ChatList friends={friends} />
+              </Route>
+              {friends.map((friend) => (
+                <Route exact path={`/chat/${friend.id}`}>
+                  <Chat friend={friend} user={user} />
+                </Route>
+              ))}
+              <Route exact path={`/profile`}>
+                <Profile user={user} />
+              </Route>
```

Comment:
```suggestion
              <Route exact path={`/`}>
                <Home friends={friends} />
              </Route>
              <Route path={`/chatlist`}>
                <ChatList friends={friends} />
              </Route>
              {friends.map((friend) => (
                <Route path={`/chat/${friend.id}`}>
                  <Chat friend={friend} user={user} />
                </Route>
              ))}
              <Route path={`/profile`}>
                <Profile user={user} />
              </Route>
```

`'/'`를 제외하면 url이 포함관계를 갖지 않으니까 굳이 `exact`를 추가하지 않아도 됩니다.